src/models/shopSchema.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose')
+
+const ShopSchema = mongoose.Schema({
+    id: Number,
+    name: String,
+    type: String,
+    location: {
+        postcode: String,
+        town: String
+    }
+})
+
+module.exports = mongoose.model('Shop', ShopSchema)
```

Comment:
Normally you export the ShopSchema, rather than the model

Reply:
I have now changed this to export the schema and create a model within the shop controller (from the schema) and then create an instance of the model (which is used within the HTTP calls)

Reply:
Have edited this so the module exports the schema and the model is created in the shops controller